terraform/weblogic_variables.tf

```diff
@@ -195,4 +195,4 @@ variable "deploy_sample_app" {
   type        = bool
   description = "Set to true to install a sample application in the WebLogic domain"
   default     = true
-}
\ No newline at end of file
+}
```

Comment:
looks like nothing changed in this file, except a white space. If possible, I suggest to remove this change.

Reply:
ok